home/context_processors.py

```diff
@@ -0,0 +1,5 @@
+from django.conf import settings
+
+
+def google_analytics_key(request):
```

Comment:
Pergunta: essa abordagem cobre as páginas do `django-admin` também?

Reply:
Dessa forma o parâmetro com a chave do Analytics fica disponível em qualquer requisição, porém é preciso adicionar o snippet com o código JS dele nas páginas que quisermos ativar. Vou criar um template base aqui para o que for do admin, e faço os testes nessas páginas também.

Uma outra coisa que preciso corrigir é ele só incluir quando estiver em Prod.